.github/workflows/ci.yml

```diff
@@ -25,3 +25,6 @@ jobs:
           bundler-cache: true
       - run: bundle exec rubocop
       - run: bundle exec rspec
+      - name: E2E tests
+        if: ${{ success() && matrix.os != 'macos-10.15' }}
```

Comment:
We can't run E2E tests on Mac because it doesn't come with docker, see https://github.com/actions/virtual-environments/issues/17.